protocol/contact.go

```diff
@@ -363,16 +364,8 @@ func BuildContactFromPublicKey(publicKey *ecdsa.PublicKey) (*Contact, error) {
 	return buildContact(id, publicKey)
 }
 
-func getShortenedCompressedKey(publicKey string) string {
-	if len(publicKey) > 9 {
-		firstPart := publicKey[0:3]
-		ellipsis := "..."
-		publicKeySize := len(publicKey)
-		lastPart := publicKey[publicKeySize-6 : publicKeySize]
-		abbreviatedKey := fmt.Sprintf("%s%s%s", firstPart, ellipsis, lastPart)
-		return abbreviatedKey
-	}
-	return ""
+func getShortenedCompressedKey(compressedKey string) string {
```

Comment:
It seems that we can delete this function, which has only one usage and directly use `alias.ShortenedCompressedKey(compressedKey)`